### PR TITLE
Patch pipelines release v0.2 once more

### PR DESCRIPTION
**Pull request by simonbaird**
This is the same as PR #1411 but hopefully with the correct target branch. :sweat_smile: .

### CODE REVIEW

File: .tekton/verify-enterprise-contract-task-v02-push.yaml [hunk 1]
```diff
@@ -10,10 +10,10 @@ metadata:
       == "main"
```
Comment (joejstuart) [inline .tekton/verify-enterprise-contract-task-v02-push.yaml:10]: should be release-v0.2

File: .tekton/cli-v02-pull-request.yaml [hunk 2]
```diff
@@ -8,24 +8,24 @@ metadata:
...
-  name: cli-main-ci-on-pull-request
+  name: cli-v02-on-pull-request
   namespace: rhtap-contract-tenant
 spec:
   params:
   - name: dockerfile
     value: Dockerfile.dist
   - name: git-url
-    value: '{{repo_url}}'
+    value: '{{source_url}}'
```
Comment (simonbaird) [inline .tekton/cli-v02-pull-request.yaml:24]: I've got another commit to fix this in main branch..

File: .tekton/verify-enterprise-contract-task-v02-push.yaml (hunk 1 above)
Comment (simonbaird) [reply .tekton/verify-enterprise-contract-task-v02-push.yaml]: Good catch, thanks.

File: .tekton/cli-v02-pull-request.yaml (hunk 2 above)
Comment (simonbaird) [reply .tekton/cli-v02-pull-request.yaml:24]: https://github.com/enterprise-contract/ec-cli/pull/1419

**Pull request merged**